routes/recommend_routes.py

```
@@ -1,80 +1,54 @@
-            "user_id" : user_id,
-            "matched_programs": matched_list
-        },
-        status_code=status.HTTP_200_OK
-    )
+
+
+    return matched_list
 
 @recommend_router.post("/{user_id}")
-def save_program(user_id: int, body: ScheduleRequest):
+def save_program(user_id : int, body: ScheduleRequest, db: Session=Depends(get_db)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)